package.json

```diff
@@ -1751,21 +1751,20 @@
             "description": "%configuration.razor.languageServer.debug%",
             "order": 90
           },
-          "razor.trace": {
+          "razor.server.trace": {
```

Comment:
why is this still called trace when it is actually log level?

why is it not named `razor.server.loglevel`?

Reply:
Just following the same convention as Roslyn. I believe its a VS Code convention too.

Reply:
that's very confusing

Reply:
Agree it's confusing 🤷

Reply:
See https://code.visualstudio.com/api/language-extensions/language-server-extension-guide#logging-support-for-language-server

Reply:
Does that actually change anything that vscode controls? It seems like basically another config to me.

Reply:
If you specify it exactly as they describe there, this setting will output the full json trace logs (when enabled) to the trace output channel you pass into the vscode LSP client.

Since we added that option though for roslyn, we've modified it a bit, and now we have to manually convert our option values and manually set the trace setting on the LSP client.  We do that here - https://github.com/dotnet/vscode-csharp/blob/main/src/lsptoolshost/roslynLanguageServer.ts#L137

Reply:
Interesting. So Roslyn (and with this PR, Razor) have `<language>.server.trace`, but the doco says `<language>.trace.server`. So, should we be consistent with Roslyn, or with VS Code?

Reply:
I guess it makes sense to be consistent with Roslyn (for me).  We use `dotnet.server.trace` because
1.  Our configs have a common prefix of `dotnet`
2.  Our configs for server related configurations are `dotnet.server.xyz`
3.  Its tracing configuration, so `dotnet.server.trace`

Reply:
> Since we added that option though for roslyn, we've changed the set of values you can use, and now we have to manually convert our option values

Yep, this is exactly what this PR is doing too, so I agree it makes sense for the setting name to be consistent if the options are.